packages/main/src/components/AnalyticalTable/types/ColumnType.ts

```diff
@@ -1,10 +1,11 @@
-import { ComponentType } from 'react';
+import { ComponentType, RefObject } from 'react';
 import { Column } from 'react-table';
+import { Ui5PopoverDomRef } from '../../../interfaces/Ui5PopoverDomRef';
```

Comment:
should we import from `@ui5/webcomponents-react/interfaces/UI5PopoverDomRef` or do we not care in this case?

Reply:
fixed